src/plugins/hetero/src/plugin.cpp

```diff
@@ -77,23 +81,101 @@ ov::hetero::Plugin::DeviceProperties ov::hetero::Plugin::get_properties_per_devi
     return device_properties;
 }
 
+bool ov::hetero::Plugin::get_device_memory_map(const std::vector<std::string>& device_names,
```

Comment:
Please add comments for TODO. There is no unified API to get device memory. So this feature get memory of specific device with specific method.

Reply:
Updated, thanks.